src/update/NoUpdatesAvailableDialog.cpp

```diff
@@ -0,0 +1,95 @@
+/**********************************************************************
+
+  Audacity: A Digital Audio Editor
+
+  NoUpdatesAvailableDialog.cpp
+
+  Anton Gerasimov
+
+**********************************************************************/
+
+#include "NoUpdatesAvailableDialog.h"
+
+#include "widgets/HelpSystem.h"
+#include "ShuttleGui.h"
+
+#include "prefs/PrefsDialog.h"
+#include "ui/AccessibleLinksFormatter.h"
+
+#include <wx/stattext.h>
+
+static const auto title =
+/* i18n-hint: Title of the dialog no updates available. */
+XO("No updates available");
```

Comment:
Maybe this should capitalize every word?

Reply:
In current design it should without every word capitalization.

Reply:
Designs can be mistaken.

Reply:
Okay, I will write Jouni to rechecking it now.

Reply:
Title Case vs Sentence case for titles... Americans like the former, in the UK the latter is more common. If Audacity already has an established title style using Title Case, then let's go with that

Reply:
@petersampsonaudacity can you remind us where "Connie" wrote down her guidelines?

Reply:
> 
> 
> @petersampsonaudacity can you remind us where "Connie" wrote down her guidelines?

https://alphamanual.audacityteam.org/man/Consistency#Capitalization_of_page_title_and_section_headings

Reply:
thanks @petersampsonaudacity !

So sentence case it is for titles

Reply:
> thanks @petersampsonaudacity !
> 
> So sentence case it is for titles

Well really the tanks for that actually go to Gale Andrews (RIP)

Reply:
Ah, so this was really enforced for code and in fact the opposite convention, "Title Case" not "Sentence case", actually prevails, as you can see in manual images, or by running the program, or by scanning code for `wxDialogWrapper` and seeing the strings passed to its constructor.

So I say "Title Case" is right, to be consistent with other dialogs.  Consistency with manual pages is the wrong criterion.

Reply:
Examples that exist now
```
VST Effect Options
Audio Unit Effect Options
Import Audio Unit Presets
LADSPA Effect Options
LV2 Effect Settings
Select Preset
Save Preset
Manage Curves List
Command Output
Export Multiple
Import Raw Data
Audacity Preferences
Label Track Font
Set Rate
Recording Meter Options
Playback Meter Options
Audacity Configuration Error
Automatic Crash Recovery
Select Command
Locate FFmpeg
Compact Project
Sound Activated Record
Welcome to Audacity!
Edit Genres
Audacity Timer Record
```

counterexamples
```
Invalid sample rate
Configure custom FFmpeg options
FFmpeg not found
Edit Labels

```

Reply:
> So I say "Title Case" is right, to be consistent with other dialogs. Consistency with manual pages is the wrong criterion.

It's your call now _(yours and @Tantacrul 's anyway)_ - Gale-the-Enforcer is no longer with us.

For me the important thing would be consistency
a) The app should be consistent within itself
b) the manual should be consistent within itself
c) the two can be different if that is what you want

Reply:
I propose:
* this new dialog follows the convention of most others, which is "Title Case"
* Connie should say explicitly that "Sentence case" applies to manual pages but the opposite applies to title dialogs
* Open an issue to fix the few inconsistent dialogs